src/widgets/tabs/tabs.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+import styles from './tabs.module.css';
+
+export interface Tab {
+  label: string;
+  content: React.ReactNode;
+}
+
+interface TabsProps {
+  tabs: Tab[];
+}
+
+export function Tabs({ tabs }: TabsProps) {
```

Comment:
```suggestion
export function Tabs({ tabs }: { tabs: Tab[] }) {
```
Unnecessary interface.